tools/dataform_assertions/customer_data/customer.csv

```diff
@@ -0,0 +1,11 @@
+customerId, first_name, last_name, second_last_name
+1, "Test1", "Specimen1", "Specimen1"
+2, "Test2", "Specimen2", "Specimen2"
+3, "Tes+t3", "Specimen", "Specimen3"
+4, "Test4", "Specimen", "Specimen4"
+5, "Test5", "Specimen", "Specimen5"
+6, "Test6", "Specimen", "Specimen6"
+7, "Tes$t7", "Specimen", "Specimen7"
+8, "Test8", "Specimen", "Specimen8"
+9, "Testtt9", "Specimen", "Specimen9"
+10, "Alphayyyiu", "Specimen", "Specimen10"
```

Comment:
We don't need these files once we change our sources to be Views with the same static data